research.qmd

```diff
@@ -2,9 +2,14 @@
 title: "Research"
 ---
 
-### Peer-Reviewed Publications
+## Peer-Reviewed Publications
+
+### 2024
+
 Cliff, B.Q., [Eddelbuettel, J.C.P.]{.underline}, Meiselbach, M.K., & Eisenberg, M.D. Deductible Imputation in Administrative Medical Claims Datasets. *Health Serv Res*. 2024; 1-7. [doi:10.1111/1475-6773.14278](https://doi:10.1111/1475-6773.14278)
 
+### 2023
+
 Shen, K., [Eddelbuettel, J.C.P.]{.underline}, & Eisenberg, M.D. The Impact of the COVID-19 Pandemic on Job Flows Into and Out of Healthcare. *JAMA Health Forum*. 2024; 5(1): e234964. [doi:10.1001/jamahealthforum.2023.4964](https://doi:10.1001/jamahealthforum.2023.4964)
```

Comment:
We should also move this paper to the 2024 section.

Reply:
Will do.